pkg/client/core/service_monitoring_client.go

```diff
@@ -51,13 +53,136 @@ const (
 	adminPassName    = "admin-pass"
 	secretsCfgName   = "grafana-secrets-cm"
 
-	lokiDatasourceConfigMapName = "loki-datasource"
+	lokiDatasourceConfigMapName  = "loki-datasource"
+	tempoDatasourceConfigMapName = "tempo-datasource"
 )
 
 func grafanaDomain(baseDomain string) string {
 	return fmt.Sprintf("%s.%s", grafanaSubDomain, baseDomain)
 }
 
+func (s *monitoringService) DeleteTempo(ctx context.Context, opts client.DeleteTempoOpts) error {
+	err := s.spinner.Start("Tempo")
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		err = s.spinner.Stop()
```

Comment:
Should probably be `_ = s.spinner.Stop()` since we're not handling the error

Reply:
The idea, at some point in time, was to override the named return, but we should probably not be failing due to a failed spinner. So yes, agreed!

Reply:
I was thinking more along the lines of if its `err =`, then a dev will have to look for when we do something depending on it. With `_ =` we dont have to look. Saves us approx 4 seconds.